### PR TITLE
Added RSS implementation

### PR DESCRIPTION
**Pull request by SobriKeN**
No merge conflicts, which is good

### CODE REVIEW

File: compose/clitools/Dockerfile [hunk 1]
```diff
@@ -28,6 +28,11 @@ RUN git clone https://github.com/stjiris/backup-jurisprudencia && cd backup-juri
 #   At 10:00 on Sunday.
 RUN (crontab -l; echo "0 10 * * 0 cd /home/clitools/backup-jurisprudencia && /usr/local/bin/node cli backup jurisprudencia.12.0,users.0.0,keys-info.0.0 >> ../backup-jurisprudencia.full.stdout.log 2>> ../backup-jurisprudencia.full.stderr.log") | crontab -
 
+# update rss feed
+RUN git clone https://github.com/SobriKeN/jurisprudencia-rss.git && cd jurisprudencia-rss && npm install && npm run build
```
Comment (diogoalmiro) [inline compose/clitools/Dockerfile:32]: Update link

File: rss.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (diogoalmiro) [inline rss.xml:1]: This file should not be commited

File: src/pages/api/rss.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { aggs } from "@/core/elasticsearch";
+import { createReadStream } from "fs";
+import { NextApiRequest, NextApiResponse } from "next";
+import path from "path";
+
+export default async function docApiHandler(
```
Comment (diogoalmiro) [inline src/pages/api/rss.ts:6]: Look at other api routes, I added a Logger.

`export default LoggerApi(async function docApiHandler` (and import LoggerApi)

File: Área Criminal.xml
Comment (diogoalmiro) [inline Área Criminal.xml:1]: Remove this file

File: Área Cível.xml
Comment (diogoalmiro) [inline Área Cível.xml:1]: Remove this file

File: Área Social.xml
Comment (diogoalmiro) [inline Área Social.xml:1]: Remove this file

**Review by diogoalmiro — CHANGES_REQUESTED** [review]
Check inline comments

**Comment by SobriKeN** [comment]
Changes made, please check

**Pull request merged**